internal/config/wrapper.go

```diff
@@ -96,6 +96,10 @@ func Load(path string, myID protocol.DeviceID) (*Wrapper, error) {
 	return Wrap(path, cfg), nil
 }
 
+func (w *Wrapper) Path() string {
```

Comment:
This is the path of the config file? I think you want the `FolderConfiguration.Path()`?